test/domain/secureMessage/BodySpec.scala

```diff
@@ -189,18 +189,28 @@ class BodySpec extends SpecBase {
   }
 
   "DateRange.apply" should {
-    "create the DateRange object with correct contents for English" in new Setup {
-      DateRange(params, Utils.englishLangKey).message mustBe "February 2021 to April 2021"
+    "create the DateRange.dateAsText object with correct contents for English" in new Setup {
```

Comment:
Please rename the test description as below and include the new assertions in the existing tests like below.
Once done , remove the two new tests

    "create the DateRange object with correct contents for English" in new Setup {
      DateRange(params, Utils.englishLangKey).dateAsText mustBe "February 2021 to April 2021"
      DateRange(params, Utils.englishLangKey).dateAsNumber mustBe "02 2021 to 04 2021"
    }

    "create the DateRange object with correct contents for Welsh" in new Setup {
      DateRange(params, Utils.welshLangKey).dateAsText mustBe "Chwefror 2021 i Ebrill 2021"
      DateRange(params, Utils.welshLangKey).dateAsNumber mustBe "02 2021 i 04 2021"
    }